src/views/clusteroverview/OverviewTab/resource-counts-card/ResourceCountsCard.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
```

Comment:
we can now import simply React no need for * as React anymore, and then name import {useState}, etc..

Reply:
I'm good with it. =) When did we start doing that? Fixed!